game_api.txt

```diff
@@ -159,6 +159,16 @@ on_rotate(pos, node, user, mode, new_param2)
 ^ use on_rotate = screwdriver.disallow to always disallow rotation
 ^ use on_rotate = screwdriver.rotate_simple to allow only face rotation
 
+Give Initial Stuff API
+----------------------
+
+give_initial_stuff.give(player)
+^ gives to the player
```

Comment:
The function doesn't check the setting - the setting is checked at load time. But yes, it should be noted that it reads the stuff variable which is set from the setting.